android/java/org/chromium/chrome/browser/settings/BraveWalletResetPreference.java

```diff
@@ -81,27 +82,27 @@ private void showBraveWalletResetDialog() {
                 R.string.brave_wallet_reset_settings_confirmation, mConfirmationPhrase));
 
         DialogInterface.OnClickListener onClickListener =
-                new DialogInterface.OnClickListener() {
-                    @Override
-                    public void onClick(DialogInterface dialog, int button) {
-                        if (button == AlertDialog.BUTTON_POSITIVE) {
-                            String inputText = input.getText().toString().trim();
-                            if (TextUtils.equals(inputText, mConfirmationPhrase)) {
-                                Log.w(TAG, "Reset");
-                                WalletNativeUtils.resetWallet(Utils.getProfile(false));
-                                KeystoreHelper.resetBiometric();
-                                Utils.setCryptoOnboarding(true);
-
-                                for (String key : WalletConstants.BRAVE_WALLET_PREFS) {
-                                    ChromeSharedPreferences.getInstance().removeKey(key);
-                                }
+                (dialog, button) -> {
+                    if (button == AlertDialog.BUTTON_POSITIVE) {
+                        String inputText = input.getText().toString().trim();
+                        if (TextUtils.equals(inputText, mConfirmationPhrase)) {
+                            Log.w(TAG, "Reset");
+                            WalletNativeUtils.resetWallet(Utils.getProfile(false));
```

Comment:
Do not use static methods like this to get the profile. See `SignInPreference` in upstream

Reply:
yeah, we can definitely do how it is inside `SignInPreference` and `MainSettings`. The main settings received `getProfile()` call recently, it didn't have it before, but right now we should use it.

Reply:
Becasue this PR has been already merged, I think it's best to track this kind of suggestions filing a new GitHub issue or fixing them directly when it's quick. So, here we go @bridiver, @SergeyZhukovsky: https://github.com/brave/brave-core/pull/28608